pkg/scheduler/objects/preemption.go

```diff
@@ -727,8 +727,8 @@ func (qps *QueuePreemptionSnapshot) IsWithinGuaranteedResource() bool {
 	if qps.Leaf && guaranteed.IsEmpty() {
 		return false
 	}
-	max := qps.GetMaxResource()
-	absGuaranteed := resources.ComponentWiseMinPermissive(guaranteed, max)
+	maxResource := qps.GetMaxResource()
```

Comment:
this change is related to IDE warnings ?

Reply:
This change is removing a call to ComponentWiseMinPermissive, which is incorrect.

Reply:
yes, it's IDE warning.
```
Warning:(730, 2) Variable 'max' collides with the 'builtin' function
```

Reply:
> This change is removing a call to ComponentWiseMinPermissive, which is incorrect.

I just do a rename by IDE, the call is still like before.

Reply:
Sorry, missed that it was showing up under the comments.